.github/release.yml

```diff
@@ -0,0 +1,18 @@
+changelog:
+  exclude:
+    authors:
+      - engine-flutter-autoroll
+      - fluttergithubbot
+    labels:
+      - passed first triage
+      - passed secondary triage
+  categories:
+    - title: Framework
+      labels:
+        - framework
+    - title: Tooling
+      labels:
+        - tool
+    - title: MacOS
+      labels:
+        - platform-mac
```

Comment:
Did you figure out if these are these processed in order? What happens to PR with `tool`, `platform-mac`, `passed first triage` and `passed secondary triage`?

Reply:
I am still working this issue, It isn't reading from the release configs the way I want yet.  Will update.